dashboard/modules/reporter/reporter_head.py

```diff
@@ -29,8 +29,8 @@ def __init__(self, dashboard_head):
 
     async def _update_stubs(self, change):
         if change.new:
-            ip, port = next(iter(change.new.items()))
-            channel = aiogrpc.insecure_channel("{}:{}".format(ip, int(port)))
+            ip, ports = next(iter(change.new.items()))
+            channel = aiogrpc.insecure_channel("{}:{}".format(ip, ports[1]))
```

Comment:
Any case that index error can happen here?

Reply:
Currently, the ports are [http_port, grpc_port]. So, It can't raise an IndexError here.